src/components/NavMenu/NavMenu.js

```diff
@@ -3,12 +3,12 @@ import './NavMenu.css';
 
 const NavMenu = () => {
   return (
-    <nav className="container__nav-menu nav-menu" aria-label="Main Navigation">
+    <nav className="container__nav-menu nav-menu" aria-label="Main Menu">
```

Comment:
I agree with this change! It's clearer!

Reply:
Mostly listening to the screen reader say "Main Navigation Navigation" was driving me nuts! 😂

Reply:
Yeah it would probably drive me nuts, too. Lol. But I am glad you made the change here.